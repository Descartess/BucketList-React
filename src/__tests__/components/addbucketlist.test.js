import React from 'react';
import renderer from 'react-test-renderer';
import AddBucketListForm from '../../components/addbucketlist';

describe('Add bucketlist snapshot', () => {
  const props = {
    onShowAddBucketList: jest.fn(),
    postBucketList: jest.fn(),
    onChangeBucketListAge: jest.fn(),
    onChangeBucketListName: jest.fn(),
    bucketlist_age: 12,
    bucketlist_name: '',
    showAddBucketList: false,
  };
  it(' Add Bucketlist form renders correctly', () => {
    const rendered = renderer.create(
      <AddBucketListForm {...props} />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
