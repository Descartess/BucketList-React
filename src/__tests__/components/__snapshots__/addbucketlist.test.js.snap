// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add bucketlist snapshot  Add Bucketlist form renders correctly 1`] = `null`;
